iocore/net/SSLUtils.cc

```diff
@@ -2010,8 +2013,9 @@ ssl_extract_certificate(const matcher_line *line_info, ssl_user_config &sslMultC
     }
   }
   if (!sslMultCertSettings.cert) {
-    Warning("missing %s tag", SSL_CERT_TAG);
-    return false;
+    // TS-4679:  It is ok to be missing the cert.  At least if the action is set to tunnel
```

Comment:
I think you should just nuke this commented code, and maybe add a comment above about the special lenience case for the tunneling (where you check if sslMultCertSettings.cert is NULL or not).